projects/laji/src/app/shared-modules/own-submissions/service/document-export.service.ts

```diff
@@ -235,7 +235,7 @@ export class DocumentExportService {
       if (field['@multipleBy']) {
         for (let i = 0; i < field['@multipleBy']; i++) {
           for (const key in field) {
-            if (field[key]['used']) {
+            if (field.hasOwnProperty(key) && field[key]['used']) {
```

Comment:
How about `field[key]?.['used']` ?

Reply:
That would not have the same effect. Idea is that hasOwnProperty prevents weird behavior if something is polluting our prototype. We could use the ES5 Object.keys, but old habits die hard (https://steemit.com/programming/@heejin/javascript-hasownproperty-in-for-in-loops)

Reply:
I was thinking in terms of readibility, and didn't consider that we'd need to take  prototype into account here. But fair enough, better be safe than sorry :)